Backend/Backend/Models/Requests/UpdateUserRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Backend.Models.Requests
+{
+    public class UpdateUserRequest<T>
+    {
+        public T Id { get; set; }
```

Comment:
Why do we need generic here? Are you going to use GUID here?

Reply:
No, i am using Hi/Lo algorigthm for id generation. I have used generic type, cause type of id can be changed in the future. But may be it is a bad idea. Should i use an Int32 here?

Reply:
Yes, it is highly unliked that we will have a dynamic field. You must have a basic class and later change it if needed

Reply:
With just ID.